modules/backend/behaviors/FormController.php

```diff
@@ -1,17 +1,16 @@
 <?php namespace Backend\Behaviors;
 
-use Db;
-use Str;
-use Lang;
-use Flash;
-use Event;
-use Redirect;
-use Backend;
 use Backend\Classes\ControllerBehavior;
-use October\Rain\Html\Helper as HtmlHelper;
-use October\Rain\Router\Helper as RouterHelper;
-use ApplicationException;
+use Backend\Facades\Backend;
 use Exception;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Event;
+use Illuminate\Support\Facades\Lang;
+use Illuminate\Support\Facades\Redirect;
+use October\Rain\Exception\ApplicationException;
+use October\Rain\Router\Helper as RouterHelper;
+use October\Rain\Support\Facades\Flash;
+use October\Rain\Support\Str;
```

Comment:
We will not be accepting these namespace changes, if you want to be able to click through to the source classes in your IDE then use https://octobercms.com/plugin/flynsarmy-idehelper. Fully specifying the namespaces to facades completely defeats the point and makes the application more brittle. An excellent example of this is the fact that Laravel removed the Input facade and we added it back, but any code that was directly referencing the Laravel version is now going to be broken in the L6 upgrade. 

@bennothommo in relation to that, do you think we can setup a class alias for the old facade to improve compatibility?